hack/install-pouch.sh

```diff
@@ -19,7 +19,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-POUCH_VERSION="1.0.0"
+POUCH_VERSION="5e2c01d0"
```

Comment:
TODO: after PouchContainer make a release, upgrade this version

Reply:
Don't worry! I remember this :-)